src/components/radioButton/RadioButton.tsx

```diff
@@ -244,19 +245,12 @@ class RadioButton extends PureComponent<Props, RadioButtonState> {
   }
 
   render() {
-    const {onPress, onValueChange, contentOnRight, ...others} = this.props;
+    const {onPress, onValueChange, contentOnRight, style, ...others} = this.props;
     const Container = onPress || onValueChange ? TouchableOpacity : View;
 
     return (
       // @ts-ignore
-      <Container
-        row
-        centerV
-        activeOpacity={1}
-        {...others}
-        onPress={this.onPress}
-        {...this.getAccessibilityProps()}
-      >
+      <Container row centerV activeOpacity={1} {...others} onPress={this.onPress} {...this.getAccessibilityProps()}>
```

Comment:
Pass a new prop (`containerStyle`) to the Container's `style` 
It will fix your issue and still allow styling the button container view.

Reply:
Thanks for reviewing! It's indeed a good idea to allow styling of both elements. I've added a `containerStyle` prop. Would this be good?